src/store/modules/plex/actions.js

```diff
@@ -93,25 +177,26 @@ export default {
                   tempConnection[key] = connection[key];
                 }
                 tempConnectionsArray.push(tempConnection);
-                if (connection.local === '1' && connection.uri.indexOf('plex') > -1) {
+                if (connection.local === '1' && connection.uri.indexOf('plex') > -1 && connection.uri.indexOf('com') > -1) {
                   const rawConnection = new PlexConnection();
                   Object.assign(rawConnection, connection);
                   rawConnection.uri = `${connection.protocol}://${connection.address}:${connection.port}`;
                   rawConnection.isManual = true;
                   tempConnectionsArray.push(rawConnection);
                 }
+                console.log(tempConnectionsArray)
               }
             }
             if (device.provides.indexOf('player') !== -1) {
```

Comment:
Change this if/else to be
```javascript
            if (device.provides.indexOf('server') !== -1) {
              // This is a Server
              // Create a new PlexServer object
              const tempServer = new PlexServer();
              for (const key in device) {
                tempServer[key] = device[key];
              }
              // Push a manual connection string for when DNS rebind doesnt work
              tempServer.plexConnections = tempConnectionsArray;
              if (tempServer.accessToken == null) {
                tempServer.accessToken = state.user.authToken;
              }

              dispatch('PLEX_ADD_SERVER', tempServer);
            }
```

The first line could probably just be this, but I haven't tested it
```javascript
            if (device.provides === 'server') {
```

Reply:
I didn't highlight the lines correctly. This pertains to the if/else starting with `if (device.provides.indexOf('player') !== -1) {`

Reply:
done